microreactors/mrad/steady_Na/HPMR_sockeye_ss.i

```diff
@@ -0,0 +1,262 @@
+  []
+[]
+
+[SolidProperties]
+  [sp_ss316]
+    type = ThermalSS316Properties
+  []
+[]
+
+[Components]
+  [heat_pipe]
+    type = HeatPipeVO
+
+    position_evap_end = '0 0 0'
+    direction_evap_to_cond = '0 0 1' # to be compatible with the core mesh
+    gravity_vector = '9.8 0 0' # assume the core is horizontal
+
+    L_evap = ${length_evap}
+    L_adia = ${length_adia}
+    L_cond = ${length_cond}
+
+    D_clad_o = ${D_clad_o}
+    D_clad_i = ${D_clad_i}
+    D_wick_o = ${D_wick_o}
+    D_wick_i = ${D_wick_i}
+
+    porosity = ${porosity}
+    permeability = ${permeability}
+    R_pore = ${R_pore}
+
+    n_elems_evap = ${n_elems_evap}
+    n_elems_adia = ${n_elems_adia}
+    n_elems_cond = ${n_elems_cond}
+
+    n_elems_wick = ${n_elems_wick}
+    n_elems_ann  = ${n_elems_ann}
+    n_elems_clad = ${n_elems_clad}
+
+    initial_T = ${T_ext_cond}
+
+    fill_ratio = ${fill_ratio}
+    T_ref_fill_ratio = ${T_ref_fill_ratio}
+    T_ref_rho_wick = ${T_ref_rho_wick}
+    T_ref_rho_clad = ${T_ref_rho_clad}
+
+    fp_2phase = fp_2phase
+    sp_wick = sp_ss316
+    sp_clad = sp_ss316
+    closures = closures
+
+    slope_reconstruction = NONE
+    stop_vapor_at_condenser_pool = true
+    startup_front_option = continuum_flow
+    D_collision = 0.362e-9
+    Kn_transition = 0.01
+  []
+
+  [evaporator_boundary]
+    type = HSBoundaryExternalAppConvection
+    boundary = 'heat_pipe:hs:evap:outer'
+    hs = heat_pipe:hs
+    T_ext = virtual_Text
+    htc_ext = virtual_htc
+  []
+
+  # This works but is simplified
+  # Improvement needed in the future
+  [condenser_boundary]
+    type = HSBoundaryAmbientConvection
+    boundary = 'heat_pipe:hs:cond:outer'
+    hs = heat_pipe:hs
+    T_ambient = ${T_ext_cond}
+    htc_ambient = ${htc_ext_cond}
+  []
+[]
+
+[UserObjects]
+  [surf_T]
+    type = LayeredSideAverage
+    direction = z
+    num_layers = 180
+    variable = T_solid
+    boundary = 'heat_pipe:hs:evap:outer'
+  []
+[]
+
+[AuxKernels]
+  [hp_var]
+    type = SpatialUserObjectAux
+    variable = hp_temp_aux
+    user_object = surf_T
+  []
+  [virtual_Text]
+    type = ParsedAux
+    variable = virtual_Text
+    coupled_variables = 'T_solid master_flux virtual_htc'
+    expression = 'master_flux/virtual_htc + T_solid'
+    block = '3 4 5'
+  []
+[]
+
+
+[AuxVariables]
+  [T_wall_var]
+    initial_condition = ${T_ext_cond}
+  []
+  [operational_aux]
+    initial_condition = 1
+  []
+  [master_flux]
+    initial_condition = 0
+  []
+  [master_t_solid]
+    initial_condition = ${T_ext_cond}
+  []
+  [hp_temp_aux]
+    initial_condition = ${T_ext_cond}
+  []
+  [virtual_Text]
+    initial_condition = ${T_ext_cond}
+  []
+  [virtual_htc]
+    initial_condition = 1.0
+  []
+[]
+
+[Postprocessors]
+  [refill]
+    type = FunctionValuePostprocessor
+    function = '1'
+    execute_on = 'INITIAL TIMESTEP_END'
+  []
+[]
+
+[Preconditioning]
+  [pc]
+    type = SMP
+    full = true
+  []
+[]
+
+# Added dampers as some NAN behavior was observed
```

Comment:
might want to explain this a little more or if Josh has some tips on how to avoid this